src/plot.jl

```diff
@@ -0,0 +1,448 @@
+if we had an infinite number of procs and the real time.
+"""
+function boundsplot(ax, loginfo::LoggerInfo)
+    # Axis attributes
+    # ---------------
+    ax.xticks = (1:3, ["Critical\nPath", "Without\nWaiting" , "Real"])
+    ax.ylabel = "Time (s)"
+
+    # Barplot
+    # -------
+    barplot!(
+        ax,
+        1:3,
+        [loginfo.t∞, loginfo.t_nowait , (loginfo.lasttime-loginfo.firsttime)],
+        color = cgrad(:sun)[1:3]
+    )
+end
+
+"""
+
+"""
+function categoriesplot(ax, loginfo::LoggerInfo)
+    categories = loginfo.categories
+    hasdefault = (loginfo.timespercat[end] !=0 ? true : false) 
+
+    # Axis attributes
+    lengthx = length(categories)
+    ticks = categories
+    hasdefault && (lengthx += 1)
+    hasdefault && (ticks = [ticks..., "default"])
+    ax.xticks = (1:lengthx, ticks)
+    ax.ylabel = "Time (s)"
+
+
+    # Colors
+    grad = cgrad(:tab10)[1:length(categories)]
+    hasdefault && (grad = [grad..., :black])
+
+    # Barplot
+    # -------
+    y = loginfo.timespercat[1:end]
+    !hasdefault && (y = y[1:end-1])
+    barplot!(
+        ax,
+        1:lengthx,
+        y,
+        color = grad
+    )
+
+end
+
+
+"""
+    dagplot(ax, logger)  
+Plot the dag to the Makie axis ax
+"""
+function dagplot(ax, logger)
+    # Create GraphViz graph DOT format file
+    g = GraphViz.Graph(loggertodot(logger))
+
+    # Node positionning
+    GraphViz.layout!(g)
+
+    # Render this to a Cairo context to convert in png
+    cs = GraphViz.cairo_render(g)
+    Cairo.write_to_png(cs, "./dag_tmp.png")
+
+    # Load the png we juste created
+    img = load("./dag_tmp.png")
+
+    # Render the png on the Makie axis
+    GLMakie.image!(ax, rotr90(img))
+
+    # Axis attributes
+    ax.aspect = DataAspect()
+    hidedecorations!(ax)
+    hidespines!(ax)
+
+    # Remove the temporary png
+    rm("./dag_tmp.png")
+end
+function dagplot(logger=getlogger())
+    fig = Figure()
+    dagplot(Axis(fig[1,1], title = "Graph"), logger)
+    fig
+end
+
+
+function react(ax, logger::Logger, gantt::Gantt)
+    to = Observable("")
+
+    on(events(ax.parent).mouseposition) do mp
+        pos = mouseposition(ax.scene)
+
+        k = 1
+        match = false
+        for tasklog ∈ Iterators.flatten(logger.tasklogs)
+            condx = gantt.starts[k] <= pos[1] <= gantt.stops[k]
+            condy = gantt.threads[k] - 0.3 <= pos[2] <= gantt.threads[k] + 0.3
+            if condx && condy
+                match = true
+                tasklog.label != "" && (to[] = "$(tasklog.label)")
+            end
+
+            k += 1
+        end
+
+        !match && (to[] = "Task Label")
+
+    end
+    on(to) do t
+        ax.title = "Parallel Trace\n$t"
+    end
+end
+
+
+"""
+    plot(logger; categories)  
+Plot DataFlowTasks `logger` labeled informations with categories.  
+Note : if there's too many tasks, the DAG won't be plotted,
+although you can manually call `dagplot(logger)`.
+
+## Example  
+Use the above exemple with
+```jldoctest
```

Comment:
I did realise that afterwards (wednesday). It's the first time i use this documentation system, it was a first draft, i'll work on that.

Reply:
OK. Now that this docstring is a simple example (not a doctest anymore), would it make sense to add a call to `plot` in one of the "real" test cases. I'm not sure we can test that the result is correct, but it might be nice to at least making sure the code is runnable without errors?